### DISCUSSION TITLE
It has been a week since I was verified as a student on Nov 27, 2023, but I cannot get my benefits now.

### DISCUSSION THREAD

**Discussion by FantasticGNU** [community/community#77785] [GitHub Education]
### Hi everyone,

It has been a week since I was verified as a student on Nov 27, 2023, but I cannot get my benefits now.

**Comment by erickdc7** [comment]
Hello,

Typically, after your academic affiliation is verified, you should gain access to GitHub Global Campus immediately, and your educational benefits should arrive within **72 hours** of this verification. However, during busy periods, such as the start of a new semester, it may take a bit longer to receive these benefits.

If it has been a week since your verification and you still haven't received your benefits, here's what you can do:

1. **Check Your Email**: Look for an email notification from GitHub regarding your benefits. Make sure to check your spam or junk folder as well.

2. **Check Your Status**: You can check the status of your benefits on the GitHub Education website.

3. **Contact GitHub Support**: If you do not receive a notification after 72 hours of your approved status, you should contact GitHub Support for assistance. They can help ensure that you receive your benefits promptly.

Remember, as a verified GitHub Global Campus student, you can explore the platform, connect with a local Campus Expert, attend events hosted by student leaders and GitHub Education, and much more while you wait for your benefits to be activated.
